Modules/Relationships/src/Relationships.Application/Relationships/Commands/CreateRelationship/Handler.cs

```diff
@@ -67,7 +67,8 @@ private async Task CreateAndSaveRelationship()
 
         var relationships = existingRelationships.ToList();
 
-        EnsureNoTerminatedRelationshipExists(relationships);
+        foreach (var relationship in relationships)
+            relationship.EnsureNotStatus(RelationshipStatus.Terminated);
```

Comment:
And please remember to add corresponding tests for the constructor then.

Reply:
Ok, I've removed the `NotInStatus` method then and added this so that it corresponds to the other `Ensure` we have:
```csharp
    private static void EnsureNoRelationshipInStatus(RelationshipStatus status, List<Relationship> existingRelationships)
    {
        if (existingRelationships.Any(r => r.Status == status))
            throw new DomainException(DomainErrors.RelationshipIsInIncorrectStatus(status));
    }
```

Changes made.